### PR TITLE
Adding mongodb container

### PR DESCRIPTION
**Pull request by ashleycoles**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -20,7 +20,18 @@ services:
...
+
+  mongodb:
+    image: mongo:latest
+    environment:
+      MONGO_INITDB_ROOT_USERNAME: root
+      MONGO_INITDB_ROOT_PASSWORD: password
+    ports:
+      - 27017:27017
+    volumes:
+      - /data/db
```
Comment (mporam) [inline docker-compose.yml:32]: can this be run from a different folder? with the mysql one we store the data in `./db` - it makes the docker image portable

File: docker-compose.yml [hunk 2]
```diff
@@ -20,7 +20,18 @@ services:
     environment:
       MYSQL_ROOT_PASSWORD: password
     restart: unless-stopped
- 
+
+  mongodb:
+    image: mongo:latest
```
Comment (mporam) [inline docker-compose.yml:25]: should we fix this at a version we know works?

File: docker-compose.yml (hunk 1 above)
Comment (ashleycoles) [reply docker-compose.yml]: Done!

File: docker-compose.yml (hunk 2 above)
Comment (ashleycoles) [reply docker-compose.yml]: Done!

**Pull request merged**